artemis-commons/src/main/java/org/apache/activemq/artemis/api/core/QueueConfiguration.java

```diff
@@ -339,6 +339,13 @@ public SimpleString getAddress() {
       return address == null ? getName() : address;
    }
 
+   /**
+    * {@return the actual qname of the address}
+    */
+   public SimpleString getActualAddress() {
```

Comment:
What about something simpler like `isAddressNull`, e.g.:
```java
public boolean isAddressNull() {
   return address == null;
}
```
I think that would be more straight-forward.

Reply:
good suggestion, does what it says on the tin! +1 for that

Reply:
method name updated